Update venice

Claude477 opened the pull request:
Remove bottom text in map

grilledham reviewed (changes requested):
We could crop the image to remove the water at the bottom.
Also can you commit the new `map_gen/data/.source_images/Venice.png`

grilledham committed:
Remove credits from venice map.

For the crashsite venice map biters would spawn on the credits which is not desirable.
